docs/user/04-metrics.md

```diff
@@ -1,61 +1,71 @@
-5. The gateway enriches all received data with typical metadata of the source by communicating with the Kubernetes APIServer. Furthermore, it filters data according to the pipeline configuration.
-6. The `MetricPipeline` resource specifies the target backend for the metric gateway.
+5. The gateway discovers the metadata and enriches all received data with typical metadata of the source by communicating with the Kubernetes APIServer. Furthermore, it filters data according to the pipeline configuration.
+6. The `MetricPipeline` resource generates the config for the gateway, which specifies the target backend for the metric gateway.
 7. The backend can run within the cluster.
 8. If authentication has been set up, the backend can also run outside the cluster.
-9. The metric data is consumed using the backend system.
+9. You can analyze the metric data with your preferred backend system.
 10. The self monitor observes the metrics flow to the backend and reports problems in the MetricPipeline status.
 
 ### Metric Gateway
 
-In a Kyma cluster, the metric gateway is the central component to which all components can send their individual metrics. The gateway collects, enriches, and dispatches the data to the configured backend. For more information, see the [Gateway documentation](./gateways.md).
+In a Kyma cluster, the metric gateway is the central component to which all components can send their individual metrics. The gateway collects, enriches, and dispatches the data to the configured backend. For more information, see [Telemetry Gateways](./gateways.md).
 
 ### Metric Agent
 
-If a MetricPipeline configures a feature in the `input` section, an additional DaemonSet is deployed acting as an agent. The agent is also based on an [OTel Collector](https://opentelemetry.io/docs/collector/) and encompasses the collection and conversion of Prometheus-based metrics. Hereby, the workload puts an `prometheus.io/scrape` annotation on the specification of the Pod or service, and the agent collects it. The agent pushes all data in OTLP to the central gateway.
+If a MetricPipeline configures a feature in the `input` section, an additional DaemonSet is deployed acting as an agent. The agent is also based on an [OTel Collector](https://opentelemetry.io/docs/collector/) and encompasses the collection and conversion of Prometheus-based metrics. Hereby, the workload puts a `prometheus.io/scrape` annotation on the specification of the Pod or service, and the agent collects it. The agent pushes all data in OTLP to the central gateway.
 
 ### Telemetry Manager
 
-The MetricPipeline resource is managed by Telemetry Manager, a typical Kubernetes [operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) responsible for managing the custom parts of the OTel Collector configuration.
+The MetricPipeline resource is managed by Telemetry Managr, which is responsible for managing the custom parts of the OTel Collector configuration.
```

Comment:
Ts ts ts, a typo from you, I thought you are close to perfect? :)
```suggestion
The MetricPipeline resource is managed by Telemetry Manager, which is responsible for managing the custom parts of the OTel Collector configuration.
```

Reply:
🫨